scrapy_playwright/page.py

```diff
@@ -8,10 +10,12 @@ class PageMethod:
     """
     Represents a method to be called (and awaited if necessary) on a
     Playwright page, such as "click", "screenshot", "evaluate", etc.
+
+    If a callable is received, it will be called with the page as its only argument.
     """
 
-    def __init__(self, method: str, *args, **kwargs) -> None:
-        self.method: str = method
+    def __init__(self, method: Union[str, Callable[[Page], Any]], *args, **kwargs) -> None:
+        self.method: Union[str, Callable[[Page], Any]] = method
```

Comment:
Had to modify the types here, Python 3.8 & 3.9 builds were failing:
```
scrapy_playwright/handler.py:40: in <module>
    from scrapy_playwright.page import PageMethod
scrapy_playwright/page.py:7: in <module>
    class PageMethod:
scrapy_playwright/page.py:13: in PageMethod
    def __init__(self, method: str | Callable, *args, **kwargs) -> None:
E   TypeError: unsupported operand type(s) for |: 'type' and '_CallableType'
```

I'll probably remove support for Python 3.8 soon and change this to `collections.abc.Callable` anyway.